attack/scripts/bashrc

```diff
@@ -61,6 +61,44 @@ info() {
 } 1>&2
 readonly -f info
 
+#
+# Starting point.
+#
+
+starting_point() {
+  # Identify the starting point mode.
+  MODE="$(sed '3,3!d' /tasks.yaml)"
```

Comment:
This would be more readable and more robust if we installed `yq` in the attack container and used that over sed

Reply:
Please, see last comment.